chart/values.yaml

```diff
@@ -152,12 +152,20 @@ exporter:
     # runAsNonRoot: true
     # runAsUser: 1000
   resources: {}
-  livenessProbe: {}
-  # livenessProbe:
-  #   httpGet:
-  #    path: /
-  #    port: 6083
-  readinessProbe: {}
+  livenessProbe:
+    httpGet:
+      path: /
+      port: 9131
+    initialDelaySeconds: 20
+    periodSeconds: 10
+    timeoutSeconds: 3
+  readinessProbe:
```

Comment:
see my comment about the effect of a readiness probe,
but I guess these here should work since they are not using the varnish itself